identity-enabler/holder-mobile-app/src/factories/serviceFactory.ts

```diff
@@ -5,18 +5,19 @@ export class ServiceFactory {
     /**
      * Store the service callbacks.
      */
-    private static readonly _services: { [name: string]: () => any } = {};
+    private static readonly _services: { [name: string]: () => unknown } = {};
+
     /**
      * Store the created instances.
      */
-    private static readonly _instances: { [name: string]: any } = {};
+    private static readonly _instances: { [name: string]: unknown } = {};
```

Comment:
why isn't this just T instead of unknown?

Reply:
Using `T` here would imply that the generic is declared at the class-level and that all services/instances be of the same type `T`. The class is intended to `register` / `get` services of arbitrary type.